doc/user_guide.md

```diff
@@ -551,6 +551,165 @@ Finally duplicate links can exist. They have no real direction. the closest matc
 |--------|-------------------|
 | ?      | Duplicate         |
 
+## XML tracing report
```

Comment:
```suggestion
## XML Tracing Report
```

We use "Associated Press" style head capitalization (https://headlinecapitalization.com/).

Reply:
DONE